### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by jpdhas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,11 +3,22 @@
 # Delivery cluster on Kubernetes POC
```
Comment (sbuliarca) [inline README.md]: Let's take this line out too, please

File: README.md [hunk 2]
```diff
@@ -36,6 +47,26 @@ docker run \
...
+kubectl cluster-info
+```
+
+The following steps has to be manually done:
+
+1. Connect to the cluster and grant admin to the default user
+```
+kubectl create clusterrolebinding add-on-cluster-admin --clusterrole=cluster-admin --serviceaccount=default:default
+```
+1. Add the new environment to the jenkins pipeline. Instructions can be found [here](https://github.com/Financial-Times/k8s-pipeline-library#what-to-do-when-adding-a-new-environment).
```
Comment (sbuliarca) [inline README.md]: Let's also mention the etcd keys need to be manually added

File: README.md [hunk 3]
```diff
@@ -36,6 +47,26 @@ docker run \
...
+1. Create/ amend the app-configs for the [upp-global-configs](https://github.com/Financial-Times/upp-global-configs/tree/master/helm/upp-global-configs/app-configs) repository. Build and deploy the global config to the new environment using this [Jenkins Job](https://upp-k8s-jenkins.in.ft.com/job/k8s-deployment/job/apps-deployment/job/upp-global-configs-auto-deploy/)
+1. [Restore](#restore-k8s-config) the config from a S3 backup or synchronize the cluster with an already existing cluster to deploy all the applications using this [Jenkins Job](https://upp-k8s-jenkins.in.ft.com/job/k8s-deployment/job/utils/job/diff-between-envs/).
+
+Note:
+* If you are re-provisioning a cluster, the restoration of the config from the S3 backup should bring the cluster healthy.
+* If you are creating a new cluster, after the restoration of the config, manual intervention is required for mongodb and kafka. Steps are detailed [here](README-app_troubleshooting.md)
+
 ##  Updating a cluster
 
 **Before updating the cluster make sure you put the initial credentials(certificates & keys) that were used when
```
Comment (sbuliarca) [inline README.md:88]: Let's mention where do these credentials can be taken from

**Review by efinlay24 — APPROVED** [review]

**Comment by jpdhas** [comment]
@sbuliarca  updated the readme with the changes you mentioned

**Review by sbuliarca — APPROVED** [review]

**Pull request merged**